src/app/(proper_react)/(onboarding)/user/welcome/View.module.scss

```diff
@@ -221,3 +221,80 @@
     }
   }
 }
+
+.stepContent {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: $spacing-md;
+
+  h1 {
+    font: $text-title-2xs;
+
+    @media screen and (min-width: $screen-md) {
+      font: $text-title-xs;
+    }
+  }
+
+  .explainerTrigger {
+    background-color: transparent;
+    border-style: none;
+    color: $color-blue-50;
+    text-decoration: underline;
+    cursor: pointer;
+
+    &:hover {
+      color: $color-blue-70;
+    }
+  }
+}
+
+.dialogContents {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-lg;
+  font: $text-body-md;
+  color: $color-grey-50;
+
+  h4 {
+    font-weight: 700;
+  }
+
+  .confirmButtonWrapper {
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    min-width: $content-xs;
+    padding-block-start: $spacing-md;
```

Comment:
Question: Are we using `padding-block-*` over `padding-top|left|bottom|right`? I'm seeing instances of both across the stylesheets. 

cc/ @Vinnl @codemist 

We don't have to answer this here (or even change it for now) but we should pick one and be consistent. I'll add this as an agenda item in tech discussion.